sentry/src/main/java/io/sentry/clientreport/AtomicClientReportStorage.java

```diff
@@ -14,25 +16,28 @@
 @ApiStatus.Internal
 final class AtomicClientReportStorage implements IClientReportStorage {
 
-  private final @NotNull Map<ClientReportKey, AtomicLong> lostEventCounts;
+  private final @NotNull LazyEvaluator<Map<ClientReportKey, AtomicLong>> lostEventCounts =
+      new LazyEvaluator<>(
```

Comment:
oh, I realized we probably have to account for multithreaded access here, so also needs a double-checked locking probably

Reply:
Do you mean here or in `LazyEvaluator`?
Should work as is, just might be slow since it'll acquire the lock for every access to the map. So we traded startup performance for runtime performance.